docs/plugins/plugin-oas/index.md

```diff
@@ -17,9 +17,7 @@ We are using [Oas](https://github.com/readmeio/oas) to convert a YAML/JSON to an
 
 The Swagger plugin also contains some classes and functions that can be used in your own plugin that needs Swagger:
 
-- For example, we have [`getReference`](https://github.com/kubb-labs/kubb/blob/main/packages/swagger/src/utils/getReference.ts) that will return the ref based on the spec.
```

Comment:
Please note that I'm removing this example because the helper doesn't exist anymore